### PR TITLE
RHDEVDOCS-4753 - Logging 5.5.5 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4753

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.5.5.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-5_{context}"]
+= Logging 5.5.5
+This release includes link:https://access.redhat.com/errata/RHSA-2022:8781[OpenShift Logging Bug Fix Release 5.5.5].
+
+[id="openshift-logging-5-5-5-bug-fixes"]
+== Bug fixes
+* Before this update, Kibana had a fixed `24h` OAuth cookie expiration time, which resulted in 401 errors in Kibana whenever the `accessTokenInactivityTimeout` field was set to a value lower than `24h`. With this update, Kibana's OAuth cookie expiration time synchronizes to the `accessTokenInactivityTimeout`, with a default value of `24h`. (link:https://issues.redhat.com/browse/LOG-3305[LOG-3305])
+
+* Before this update, vector parsed the message field when JSON  parsing was enabled without also defining `structuredTypeKey` or `structuredTypeName` values. With this update, a value is required for either  `structuredTypeKey` or `structuredTypeName` when writing structured logs to Elasticsearch. (link:https://issues.redhat.com/browse/LOG-3284[LOG-3284])
```
Comment (gabriel-rh) [inline modules/cluster-logging-rn-5.5.5.adoc:11]: Should vector be have initial caps Vector? 

https://docs.openshift.com/container-platform/4.11/logging/cluster-logging-loki.html#cluster-logging-vector-enable_cluster-logging-loki

File: modules/cluster-logging-rn-5.5.5.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+* Before this update, the `FluentdQueueLengthIncreasing`alert  could fail to fire when there was a cardinality issue with the set of labels returned from this alert expression. This updated reduces labels to only include those required for the alert. (https://issues.redhat.com/browse/LOG-3226[LOG-3226])
```
Comment (gabriel-rh) [inline modules/cluster-logging-rn-5.5.5.adoc:13]:
```suggestion
* Before this update, the `FluentdQueueLengthIncreasing` alert  could fail to fire when there was a cardinality issue with the set of labels returned from this alert expression. This update reduces labels to only include those required for the alert. (https://issues.redhat.com/browse/LOG-3226[LOG-3226]) 
```

**Review by gabriel-rh — COMMENTED** [review]
Looks good - just some minor changes required.

**Comment by kabirbhartiRH** [comment]
@libander Could you check if we need to add below bugs? I see them on 5.5.5 errata but they do not have _no-rn_ label

https://issues.redhat.com/browse/LOG-3222
https://issues.redhat.com/browse/LOG-2860
https://issues.redhat.com/browse/LOG-3131
https://issues.redhat.com/browse/LOG-3310
https://issues.redhat.com/browse/LOG-3332

**Review by gabriel-rh — APPROVED** [review]
/lgtm

File: modules/cluster-logging-rn-5.5.5.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+* Before this update, the `FluentdQueueLengthIncreasing` alert could fail to fire when there was a cardinality issue with the set of labels returned from this alert expression. This update reduces labels to only include those required for the alert. (https://issues.redhat.com/browse/LOG-3226[LOG-3226])
+
+* Before this update, Loki did not have support to reach an external storage in a disconnected cluster. With this update,  proxy environment variables and proxy trusted CA bundles are included in the container image to support these connections. (link:https://issues.redhat.com/browse/LOG-2860[LOG-2860])
+
+* Before this update, {product-title} web console users could not choose the ConfigMap that includes the CA certificate for Loki, causing pods to operate without the CA. With this update, web console users can select the ConfigMap, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3310[LOG-3310])
```
Comment (bburt-rh) [inline modules/cluster-logging-rn-5.5.5.adoc:17]:
```suggestion
* Before this update, {product-title} web console users could not choose the ConfigMap that includes the CA certificate for Loki, causing pods to operate without the CA. With this update, web console users can select the ConfigMap, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3310[LOG-3310])

- Per SSG - it should either be "config map" or "`ConfigMap` object with back ticks around `ConfigMap`, but it's not worth holding up the merge over this if you can fix it in another PR.
```

**Pull request merged**